frontend/src/app/modules/distribution/distribution.component.html

```diff
@@ -2,6 +2,9 @@
 <p>
   {{ 'frontend.de.iteratec.osm.distribution.description' | translate }}
 </p>
+
+<osm-violin-chart [dataInput]="this.results$ | async" [hidden]="!showChart"></osm-violin-chart>
```

Comment:
I can not add a comment to code which you didn't authored.
But I noticed that the result selection currently offers the possibility to select a browser:
![image](https://user-images.githubusercontent.com/25281769/70053433-be1cd780-15d5-11ea-9d99-7adc1e3212df.png)

The backend doesn't differentiate in this entity at the moment. So, it would be nice if you could disable this by removing `[showBrowserSelection]="true"` in line 25 of this file.

Reply:
Done.